### ISSUE TITLE
Fast Help hand in hand with AI

### ISSUE THREAD

**Issue by Juan6078** [Juan6078/AI_Solutions#1]
Broadly speaking, our Company consists of medical treatments caring for patients who live far from medical centers, in a way that there is no easy access to urgent services or routine checking from doctors for minor pains. The usual prescription of medicines taking care of diabetes, blood pressure, or regular headaches,....., turn out to be a great problem to these people due to the fact the specialist will arrive in a couple of hours, or painfully, when there is nothing that can be done for the patient. The headquarter has ten ambulances, 15 doctors, and 8 C-suite professionals, who try to perform their duties for these patients to be safe and sound steadily, although they are not so close to them and the assistance is not easy to be done oftentimes. That is why ‘FarHelp’(the Company) has been set since its beginning  attempting to integrate AI initiatives into digital initiatives, focusing AI’s effects on business models. As a whole, the staff thought of AI as a tool to reach existing objectives, and tying AI strategy into overall strategy.

The main activities FarHelp launched in its AI strategy can be cited as follows:_

--        **Deep Learning for actionable insights.** 

Thus, streamline radiology diagnoses,the company’s deep learning platform analyzes unstructured medical data (radiology images, blood tests, EKGs, genomics, patient medical history) to give doctors better insight into a patient’s real-time needs. 

--       **Neural Network for Clinical Trials:**

Pedicting bioactivity and identify patient characteristics for clinical trials. Atomwise’s AI technology screens between 10 and 20 million genetic compounds each day and can reportedly deliver results 100 times faster than traditional pharmaceutical companies.

- -	**Deep Learning for Targeted Treatment:****

Getting the right treatment to the right patients at the right time by using artificial intelligence to produce a better target selection and provide previously undiscovered insights through deep learning.

- -	**Own your Health:******

Complete, discreet solution for managing chronic conditions like diabetes, heart health and blood pressure, as well as weight management. Allows people to manage their conditions head-first, offering interactive coaching from real-world professionals, predictive glucose readings powered by artificial intelligence and data science, learning resources and daily tracking of readings.

This chosen decision about these cited activities is based on an attempt to align AI production and consumption. Thus, FarHelp invests in the consumption of AI by our managers who are going to consume the insights of the Data Science of Machine Learning and make managerial decisions from it. Not in vain, this approach avoids producing AI algorithms which is a less effective approach at all.

         Should the term time is taken into consideration, we guess that is worth citing that the automation the activities ready to be launched might imply,  can be impactful in the long run, both in terms of reducing costs, but also improving the workflow for the customer, or patient in this case,  and winning over future customers to the organization. This portfolio approach is one that has been used by some companies to great success. Specifically, in FastHelp we felt like we  can get things going in a short period of time and thought about how AI can be transformational. As a thought of’ quick wins’ in our Company  is that it helps companies build important skills with regard to AI, these are skills with regard to gathering data at large scales, processing the data, labeling the data. Therefore,we could  create training data sets and ultimately analyze the data and interpret the results of the analysis. 

Contrarily, if our approach on AI issues were on long term projects, these activities were leaded to be impactful and provide strategic or competitive advantage to the organization In this case, we should better rethinking entirely into a processes and not just optimizing at a point, or, in this case, the output of these four activities.

        From our background, we might admit that these brand new AI activities are going to take a long time to be built, since we are a Company that doesn't have data science in our DNA, simply hiring a few data scientists will not change that. In order to change that it takes time and quick wins and short term projects help achieve some of that because it is going to help us and see results and get experience with data gathering, labeling and analysis. 

In order not to cause some great impacts on our HR structure as a Company the adoptions of these proposed brand new activities, any suggestions have been brought up by C-suite, and they are as follows: 

Identifying a set of activities within FastHelp that can automate through machine learning, such as  activities across multiple functions. Once you've come up with a set

of several dozen activities, we then want to classify these activities into short term activities, meaning activities that can be completed in six months or long term activities for which you need a few years. Once we had come up with this classification, it is high time to  looked at the R.O.I to FastHelp from these activities based on an assessment of the return on investment of these activities and also an assessment of the data assets available within us.

Once this portfolio is set, FastHelp has to figure out where the AI team will fit within the Company’s chart. The C-suited planned  to have separate AI teams that sit in a separate group and work with other groups so that help drive transformation almost like a consulting group that is working with these other teams. Undoubtedly, where the AI team will fit within the organization chart. The dramatic challenge of the Company would be solved,inevitably.
